test/integration/basic_test.go

```diff
@@ -152,6 +152,7 @@ func (m *Minishift) commandReturnShouldBeEmpty(commandField string) error {
 func FeatureContext(s *godog.Suite) {
 	var givenArgs = flag.String("minishift-args", "", "Arguments to pass to minishift")
 	var givenPath = flag.String("binary", fmt.Sprintf("../../out/%s-amd64/minishift", runtime.GOOS), "Path to minishift binary")
```

Comment:
@hferentschik Agree with you, when I implement your suggestion above this step is not required.